Misc/NEWS.d/next/Build/2021-10-01-12-20-05.bpo-0.2ykYK2.rst

```diff
@@ -0,0 +1 @@
+Makefile: fix missing slashes
```

Comment:
It is not very informative. What exactly behavior was fixed, in which circumstances Makefile did not work correctly? Is it specific to some platforms or some actions? How can readers know whether this bug affected them?